Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -45,20 +40,6 @@ public virtual void AddShortcutHandler(IShortcutHandler handler)
             _shortcutHandlers.Add(handler);
         }
 
-        protected void SetInvalidFileNameChars(bool enableManagedInvalidFileNameChars)
-        {
-            if (enableManagedInvalidFileNameChars)
-            {
-                _invalidFileNameChars = Path.GetInvalidFileNameChars();
-            }
-            else
-            {
-                // Be consistent across platforms because the windows server will fail to query network shares that don't follow windows conventions
```

Comment:
are you sure this is an invalid use case?..

Reply:
`GetInvalidFileNameChars` should do the trick, if not it's a bug in the core lib ;).
This was probably for when we were still handling network shares ourselves.